ladder/teams/ci.yaml

```diff
@@ -0,0 +1,7 @@
+members:
```

Comment:
Where did this team come from?

Reply:
typo, it's a duplicate of `cli`. Will remove.